packages/boutique/backend/src/app.ts

```diff
@@ -132,7 +132,7 @@ export class App {
       .catch((err) => {
         logger.error('Error while trying to process pending payments')
         logger.error(err)
-        return true
+        return false
```

Comment:
why is the return needed at all?

Reply:
yes, to setup the next execution

Reply:
with returning true, it was going inside the if with immediate execution and flooding the app with errors